tracer/src/Datadog.Trace.Tools.Runner/Checks/ProcessBasicCheck.cs

```diff
@@ -127,29 +137,43 @@ public static bool Run(ProcessInfo process, IRegistryService? registryService =
             string corProfilerPathKey32 = runtime == ProcessInfo.Runtime.NetCore ? "CORECLR_PROFILER_PATH_32" : "COR_PROFILER_PATH_32";
             string corProfilerPathKey64 = runtime == ProcessInfo.Runtime.NetCore ? "CORECLR_PROFILER_PATH_64" : "COR_PROFILER_PATH_64";
 
+            AnsiConsole.WriteLine(EnvVarCheck("3", corProfilerPathKey));
+
             ok &= CheckProfilerPath(process, corProfilerPathKey, requiredOnLinux: true);
             ok &= CheckProfilerPath(process, corProfilerPathKey32, requiredOnLinux: false);
             ok &= CheckProfilerPath(process, corProfilerPathKey64, requiredOnLinux: false);
 
             string corProfilerKey = runtime == ProcessInfo.Runtime.NetCore ? "CORECLR_PROFILER" : "COR_PROFILER";
 
+            AnsiConsole.WriteLine(EnvVarCheck("4", corProfilerKey));
+
             process.EnvironmentVariables.TryGetValue(corProfilerKey, out var corProfiler);
 
             if (corProfiler != Utils.Profilerid)
             {
                 Utils.WriteWarning(WrongEnvironmentVariableFormat(corProfilerKey, Utils.Profilerid, corProfiler));
                 ok = false;
             }
+            else if (corProfiler == Utils.Profilerid)
```

Comment:
nit: we can simplify here
```suggestion
            else
```